specification/cost-management/resource-manager/Microsoft.CostManagement/preview/2018-08-01-preview/costmanagement.json

```diff
@@ -1860,13 +1860,486 @@
         }
       }
     },
+    "/providers/Microsoft.Billing/billingAccounts/{billingAccountId}/providers/Microsoft.CostManagement/alerts": {
+      "get": {
+        "tags": [
+          "Alerts"
+        ],
+        "operationId": "Alerts_ListByEnrollment",
+        "description": "List all alerts for an enrollment.",
+        "x-ms-examples": {
+          "Alert_List": {
+            "$ref": "./examples/AlertList.json"
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "#/parameters/apiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/billingAccountIdParameter"
+          },
+          {
+            "name": "$filter",
+            "description": "May be used to filter alerts by properties/definition/type, properties/definition/category, properties/definition/criteria, properties/costEntityId, properties/creationTime, properties/closeTime, properties/status, properties/source. Supported operators are 'eq','lt', 'gt', 'le', 'ge'.",
```

Comment:
@yaborens
Examples corresponding to this change?

Also, is the list of filters a closed list? If so... why not use an enum?

Reply:
same comments as above

Reply:
I see only this change in the example linked here : https://github.com/Azure/azure-rest-api-specs/pull/4818/files/1d5c0565121e5fef4897b32e78318df919406717#diff-cd2cdd86afbc1836331eaef83f213b9cR30

Is that all changes in the example for this new API?

Reply:
yes